content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -23,3 +23,118 @@ All students need to provably understand all of the following concepts:
 - getters and setters
 - inheritance and overriding and extending methods
 - composition
+
+# Python
+
+## Constructors
+The `__init__()` method is called the constructor and is always called when an object is created, generally used for instantiating an object.
+
+### Default Constructor
+A simple constructor which doesn’t accept any arguments. Its definition has only one argument which is a reference to the instance being constructed.
```

Comment:
Sorry if this seems pedantic but I'd say:
The default constructor is a simple constructor which doesn’t accept any arguments. Its definition has only one argument which is a reference to the instance being constructed.